daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/Reader.scala

```diff
@@ -120,7 +121,7 @@ object Reader extends Reader[(PackageId, DamlLf.ArchivePayload)] {
   @throws[ParseError]
   def readArchiveVersion(lf: DamlLf.ArchivePayload): LanguageMajorVersion = {
     import DamlLf.ArchivePayload.{SumCase => SC}
-    import archive.{LanguageMajorVersion => LMV}
+    import com.digitalasset.daml.lf.language.{LanguageMajorVersion => LMV}
```

Comment:
```suggestion
    import language.{LanguageMajorVersion => LMV}
```

Reply:
I knew @S11001001 was going to point it out :)
twitter's Effective Scala:
> Do not use relative imports from other packages
https://twitter.github.io/effectivescala/#Formatting-Imports

I thought that at some point everyone agreed we follow twitter rules :)

Reply:
I think you're thinking of somewhere else entirely @leo-da :)

And they're wrong in several ways.

Reply:
It was a long discussion about implicits, about a year ago :)
We actually have not agreed on anything... someone just pointed out to the Twitter's Effective Scala, saying we will do what this page tells us.